packages/graphql/src/generated/index.ts

```diff
@@ -7510,6 +7510,7 @@ export type AtomVaultDetailsFragment = {
     __typename?: 'vaults'
     positionCount: number
     totalShares: any
+    currentSharePrice: any
```

Comment:
We still have a good bit of `any` in our code gen types. As we get closer to migration we'll have to see what we need to tighten up at the schema level.